### ISSUE TITLE
Improve DDC support for built-in HDMI 2.0 ports of some newer macs

### ISSUE THREAD

**Issue by KrzysztofPrzygoda** [MonitorControl/MonitorControl#1290]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Can't DCC contol monitor over HDMI build-in port, although it's possible on Lunar app.

### Expected behavior

Flawless DCC control over built-in HDMI port.

### Anything else?

Here are specifics of the issue filed by me at BetterDisplay:
https://github.com/waydabber/BetterDisplay/issues/1363
https://github.com/waydabber/BetterDisplay/issues/1364

Lunar app has flawless DCC control over HDMI M-M.
MonitorControl app does not work on LG 27UP850-W and MacBook Pro 16 Apple M1 Max.

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 13.1
- Mac model: MacBookPro18,2
- MonitorControl version: 4.1.0
- Monitor(s): LG 27UP850-W
- Monitor Cable(s)/Connection(s): HDMI-HDMI 
- Apple Silicon/M1 (yes or no): yes, M1 Max
```

**Comment by waydabber** [comment]
Right, finally figured out what works differently with that port (took most of my day today...). I'll do some testing with various hardware and if it works fine, it will be in the next BetterDisplay release. Will hopefully implement in MC as well in a future update!

**Comment by dzirtusss** [comment]
@waydabber any technical details on the way how you made it work? Thanks in advance if you can guide something on that...

**Comment by waydabber** [comment]
Sorry @dzirtusss, you'll need to start digging yourself with this, the method is not open sourced for now. But the fact that I was able to figure it out in a day shows the task is certainly not impossible! :) If you figure it out, feel free to add a PR to MonitorControl!

**Comment by waydabber** [comment]
I closed this as it was implemented in BetterDisplay. MonitorControl's feature set is now considered final by the team, but if a PR relevant to this comes along, it will be accepted of course.
